countdown/countdown.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
```

Comment:
this is not needed in a web script, only command line node files if they're running by themselves